eng/Versions.props

```diff
@@ -136,6 +135,7 @@
     <MicrosoftVisualStudioEditorVersion>$(VisualStudioEditorNewPackagesVersion)</MicrosoftVisualStudioEditorVersion>
     <MicrosoftVisualStudioExtensibilityTestingSourceGeneratorVersion>$(MicrosoftVisualStudioExtensibilityTestingVersion)</MicrosoftVisualStudioExtensibilityTestingSourceGeneratorVersion>
     <MicrosoftVisualStudioExtensibilityTestingXunitVersion>$(MicrosoftVisualStudioExtensibilityTestingVersion)</MicrosoftVisualStudioExtensibilityTestingXunitVersion>
+    <MicrosoftVisualStudioFPFVersion>$(VisualStudioEditorNewPackagesVersion)</MicrosoftVisualStudioFPFVersion>
```

Comment:
Per [this]( https://dev.azure.com/dnceng/public/_build/results?buildId=1641923&view=logs&j=5e4c1a10-89a7-5bf1-0765-c10b6cf6bdb8&t=8490a8da-5b3c-5ac4-77a4-358fbd282c33&l=716) CI run we do not allow this naming scheme

> PackageReference Microsoft.VisualStudio.FPF.PresentationFramework has incorrect version $(MicrosoftVisualStudioFPFVersion)
> Allowed values are $(MicrosoftVisualStudioFPFPresentationFrameworkVersion) or$(MicrosoftVisualStudioFPFPresentationFrameworkFixedVersion) or$(RefOnlyMicrosoftVisualStudioFPFPresentationFrameworkVersion)

Reply:
Thanks, makes sense.

Reply:
Fixed.